README.md

```diff
@@ -11,21 +11,23 @@ Without the `<>`.
 
 Example:
 ```yaml
-name: Example name
-link: https://example.com/
+name: John Camilton
```

Comment:
maybe not use an actual example?

Reply:
I had the opinion that `Example name` might not be so clear as a real name, that's why I changed it. But that's up for discussion.